iib/workers/tasks/fbc_utils.py

```diff
@@ -17,6 +19,45 @@ def is_image_fbc(image):
     :return: True if image is FBC type, False otherwise (SQLite)
     :rtype: bool
     """
+    from iib.workers.tasks.utils import skopeo_inspect
+
     skopeo_output = skopeo_inspect(f'docker://{image}')
-    dc_image_label = 'operators.operatorframework.io.index.configs.v1'
-    return dc_image_label in skopeo_output['Labels']
+    fbc_image_label = 'operators.operatorframework.io.index.configs.v1'
```

Comment:
could we use function `get_image_labels()` from `utils.py` line #528?

Reply:
https://github.com/release-engineering/iib/pull/328

Reply:
I agree - let me address that in separate PR because it is not direct part of this PR.
Thanks!

Reply:
Are we going to have a new PR?

Reply:
yes